schema/artwork/index.js

```diff
@@ -39,6 +39,10 @@ import {
   GraphQLInt,
 } from 'graphql';
 
+const is_inquireable = ({ inquireable, acquireable }) => {
+  return (inquireable && !acquireable);
+};
```

Comment:
cc @joeyAghion might know ^.

Basically I think the question is why wouldn't Gravity's `inquireable` take into account that it shouldn't be true for e-commerce/`acquireable` works.